cli/src/commands/file/annotate.rs

```diff
@@ -51,6 +52,22 @@ pub(crate) struct FileAnnotateArgs {
         add = ArgValueCandidates::new(complete::all_revisions)
     )]
     revision: Option<RevisionArg>,
+    /// Render each line using the given template
+    ///
+    /// Run `jj log -T` to list the built-in templates.
```

Comment:
I would copy help text from non-`Commit` template (such as `cmd_file_list()`.) The built-in templates are mostly useless in this context.